iotilecore/iotile/core/utilities/async_tools/event_loop.py

```diff
@@ -765,13 +769,38 @@ def _log_future_exception(future, logger):
             logger.exception("Exception in repeating coroutine; exiting repetition: %s", cor, exc_info=True)
             raise
 
+
 def _tick_generator(period, loop):
     t1 = loop.time()
     count = 0
     while True:
         count += 1
         yield max(t1 + count * period - loop.time(), 0)
 
+
+def _check_patch_python_3_8_0():
+    """Python 3.8 on windows switches event loops but has a bug.
```

Comment:
Probably not necessary, but was there a link to the python bug/fix thread for posterity? I find it fun to read the gritty details sometimes :)

Reply:
Updated with a link: 
https://bugs.python.org/issue34679